lib/typescript/translations/translations.ts

```diff
@@ -787,6 +791,8 @@ const resources = {
       searchTemplates: 'Recherche de modèles',
 
       //Control-Center
+      disableComponent: 'Désactiver',
+      disableComponentText: 'Tu es sûre de vouloir désactiver ce composant?',
```

Comment:
nitpick: peut-être vouvoiement?

`Êtes vous sûr(e) de vouloir désactiver ce composant ?`

Reply:
I think is better to keep an informal tone. At least is what I see in many other apps. I also did an informal tone on Spanish and German